bundle/Handler/FieldType/RichText.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\OpenGraphBundle\Handler\FieldType;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\RichText\Value;
```

Comment:
Isn't this value deprecated and removed in eZ kernel v8? (`master` branch is only for v8).

Reply:
This field is add in eZ kernel v7